hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/VolumeInfoMetrics.java

```diff
@@ -153,4 +154,15 @@ public void dbCompactTimesNanoSecondsIncr(long time) {
     dbCompactLatency.add(time);
   }
 
+  /**
+   * Return the Container Count of the Volume.
+   */
+  @Metric("Returns the Container Count of the Volume")
+  public long getContainers() {
+    return this.containers;
+  }
+
+  public void setContainers(long count) {
+    this.containers = count;
+  }
```

Comment:
This class currently pulls most of its metrics from the `volume` field and does not depend on external setters except for `dbCompactLatency`. Can we do the same for this metric?